staging/src/k8s.io/apiserver/pkg/storage/watch_cache_test.go

```diff
@@ -272,10 +304,13 @@ func TestWaitUntilFreshAndList(t *testing.T) {
 func TestWaitUntilFreshAndGet(t *testing.T) {
 	store := newTestWatchCache(3)
 
+	foo := makeTestPod("foo", 2)
+	bar := makeTestPod("bar", 5)
+
 	// In background, update the store.
 	go func() {
-		store.Add(makeTestPod("foo", 2))
-		store.Add(makeTestPod("bar", 5))
+		store.Add(foo)
+		store.Add(bar)
```

Comment:
revert this and reconstruct in `makeTestStoreElement(makeTestPod("bar",5))`... sharing an object between two parts of the test makes us vulnerable to mutations that make comparisons match but check the wrong thing.

Reply:
done